packages/tasit-sdk/src/TasitSdk.test.js

```diff
@@ -189,24 +209,28 @@ describe("Decentraland", () => {
             await createOrder.waitForNonceToUpdate();
           }
 
-          mana.removeWallet();
-          land.removeWallet();
-          estate.removeWallet();
-          marketplace.removeWallet();
+          manaContract.removeWallet();
+          landContract.removeWallet();
+          estateContract.removeWallet();
+          marketplaceContract.removeWallet();
         }
       );
 
       it("should list marketplace estates sell orders (without wallet)", async () => {
         const orders = [];
-        const totalSupply = await estate.totalSupply();
+        const totalSupply = await estateContract.totalSupply();
 
         // create an array 1..N, where N = total of estates
         const allEstatesIds = [...Array(totalSupply.toNumber())].map(
           (val, key) => key + 1
         );
 
         for (let id of allEstatesIds) {
-          const order = await getEstateSellOrder(marketplace, estate, id);
+          const order = await getEstateSellOrder(
+            marketplaceContract,
+            estateContract,
+            id
```

Comment:
Even id could be worth renaming to make it clear that it's the id of ____

Reply:
Done